datasets/tf_datasets.py

```diff
@@ -0,0 +1,117 @@
+"""
+https://www.tensorflow.org/datasets
+"""
+
```

Comment:
what is this? Is this supposed to be a docstring? If yes, can you please add documentation.

Reply:
Btw, why do you want to have the documentation here much longer than in all other modules (basically non-existing)?